[CMS] Add mention about files migration to 7 to 8 migration guide (#5014)
umbraco/UmbracoDocs: 11/umbraco-cms/fundamentals/setup/upgrading/version-specific/migrate-content-to-umbraco-8.md

{% hint style="info" %}
Please be aware that this is a **content migration**. If you go to the frontend after following these steps, it will throw errors.

At this point you will have the content but nothing else.
{% endhint %}

## Step 3: Files migration

Before moving on to this step, make sure that the Umbraco 8 project is no longer running.

The following files/folders need to be copied into the Umbraco 8 project:

* `~/Views` - do **not** overwrite the default Macro and Partial View Macro files, unless changes have been made to these.
* `~/Media`
* Any files/folders related to Stylesheets and JavaScripts.
* Any custom files/folders the Umbraco 7 project uses, that aren't in the `~/Config` or `~/bin`.
* `~/App_Data/UmbracoForms` - in the case Umbraco Forms was used on the Umbraco 7 site.

**Merge the configuration files carefully** to ensure any custom settings are migrated while none of the default configurations for Umbraco 8 is overwritten.

> You will have to revisit all templates and other custom implementation to get the site up and running, as all markup will still be Umbraco 7 specific.

Reviewer: ```suggestion
You'll have to revisit all templates and custom implementations to get the site up and running, as all markup is still Umbraco 7-specific.
```
